### ISSUE TITLE
warnings when compiling in windows

### ISSUE THREAD

**Issue by jpujol** [sainteos/tmxparser#75]
I am compiling in Windows (VS2017) and I am getting warnings like
```
tmxpropertyset.h(66): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
```
Basically it shows a warning for all the functions that return `size` of a stl container, since the return value is cast to an `int`. Technically, `size()` returns a `size_type`.

I was going to create a PR changing all the interfaces that get the size of a container to return a `size_type`, but I wanted to ask for your opinion first, @sainteos

**Comment by sainteos** [comment]
Yeah, that change seems to make sense. If you still want to do this (sorry for the wildly late reply), go for it. If not, I can.

**Comment by jpujol** [comment]
oh, I can do it! thanks for the reply
